### ISSUE TITLE
Unable to import Fridge-tag

### ISSUE THREAD

**Issue by marthakeezy** [msupply-foundation/open-msupply#5350]
## What went wrong? 😲

Unable to import Fridge-tag - getting the following error

<img width="1512" alt="Screenshot 2024-11-07 at 4 07 58 PM" src="https://github.com/user-attachments/assets/6ca72ce6-86ac-4178-a3e1-23224e0c5c04">

It works fine with the Q-tag, but not the Fridge-tag!

## Expected behaviour 🤔

Should be able to import a Fridge-tag

## How to Reproduce 🔨

Steps to reproduce the behaviour:

1. Connect one of the Fridge-tags to your device
2. Go to Cold chain -> Monitoring -> Import Fridge-tag
3. Locate the file and attempt to upload
4. See error!

## Your environment 🌱

- Open mSupply Version: V2.4.0-test1
- Legacy mSupply Central Server Version: V7.18.11

- Platform: Google Chrome browser, Galaxy Tab8 Android

- Database type: SQLite3

**Comment by lache-melvin** [comment]
Hey @marthakeezy - needing a little more context...

Can you confirm it's not this issue? https://github.com/msupply-foundation/open-msupply/issues/4755 (Do you have another store that also have fridge tags imported? If so, can you import okay into that store?)

If not, can you confirm if this is only in 2.4.0-test, or also in 2.3?

**Comment by marthakeezy** [comment]
It is actually entirely possible that Fridge-tag has already been imported into another store, as we just grabbed it off Carl, however I am running SQLite3 (issue you linked mentions postgress). Also happening in `V2.3.2-RC1`

**Comment by lache-melvin** [comment]
Oh sorry, I meant in your datafile, do you have any other stores, which might have fridge tags already imported into them? Though yes, I think that is just PG...

**Comment by marthakeezy** [comment]
I do - on my remote site, I have one store with a Q-Tag and a Fridge-tag imported

**Comment by lache-melvin** [comment]
..haha, you have only one store, or another store than the one pictured, that already has the fridge tag imported?

**Comment by marthakeezy** [comment]
The one pictured in the issue is my Central Server store
I have another remote site on the same datafile, with a Q-tag and a Fridge-tag imported

**Comment by lache-melvin** [comment]
Are you able to import this fridge tag into that store?

**Comment by marthakeezy** [comment]
Nope, can't import into that store either - I initially tried on a remote site and then tried on my central server one just in case. Running into the same error on both

**Comment by andreievg** [comment]
Lowering priority, there is a linked issue with PR that potentially solves this

**Comment by DhanyaHerath** [comment]
Would you mind retesting this @marthakeezy when you have a chance?

**Comment by marthakeezy** [comment]
I think it's working now - grabbed the fridge-tag from the kitchen fridge that has been imported into one of Adam's stores before, and it successfully imported into my own :) closing, can reopen if any other issues occur!
